src/test/kotlin/no/nav/syfo/testhelper/TestDatabase.kt

```diff
@@ -44,6 +46,25 @@ fun DatabaseInterface.createNMeldingTilBehandler(meldingTilBehandler: MeldingTil
         }
         connection.commit()
     }
+    return meldingTilBehandler.conversationRef
+}
+
+fun DatabaseInterface.createNMeldingFraBehandler(meldingFraBehandler: MeldingFraBehandler, numberOfMeldinger: Int = 1): UUID {
```

Comment:
createNMeldingFraBehandler, hva står N for?

Reply:
N antall meldinger sikkert

Reply:
Hvorfor ikke bare `createMeldingerFraBehandler`?